composer.json

```diff
@@ -15,8 +15,10 @@
     ],
 
     "require": {
-        "doctrine/common": "^2.3",
+        "doctrine/annotations": "^1.12",
         "doctrine/dbal": "^2.5",
+        "doctrine/persistence": "^1.3|^2.1",
+        "doctrine/orm": "^2.0",
```

Comment:
Hast Du das bewusst zur direkten (harten) Abhängigkeit gemacht?

Reply:
Ja - beim Doctrine für SOB -> Staats-X -> Bacchus-Update bin ich mehrfach darüber gestolpert, dass Pakete ihre Abhängigkeiten nicht deklarierten und dann mit neueren Versionen z.B. von doctrine/persistence nicht klar kamen. Ich finds zwar auch doof, das zur harten Abhängigkeit zu machen, aber (inbesondere bei diesen weit verbreiteten oder in sich sehr kompakten Abhängikeiten) mir war es wichtiger auszudrücken, womit das WebfactoryWfdMetaBundle zusammenspielt.

Etwas ausführlicher, allgemeiner im commit: https://github.com/webfactory/WebfactoryWfdMetaBundle/pull/13/commits/8fbaa8fef7b0a566da141b145a51e5643f2f7eff